pull.go

```diff
@@ -263,86 +191,15 @@ func (target *Target) ReplacePlaceholders(localeFile *LocaleFile) (string, error
 	return path, nil
 }
 
-func (t *Target) GetFormat() string {
-	if t.Params != nil && t.Params.FileFormat != nil {
-		return *t.Params.FileFormat
-	}
-	if t.FileFormat != "" {
-		return t.FileFormat
-	}
-	return ""
-}
-
-func (t *Target) GetLocaleID() string {
-	if t.Params != nil {
-		return t.Params.LocaleID
-	}
-	return ""
-}
-
-func (t *Target) GetTag() string {
-	if t.Params != nil && t.Params.Tag != nil {
-		return *t.Params.Tag
-	}
-	return ""
-}
-
-func TargetsFromConfig(cmd *PullCommand) (Targets, error) {
-	if cmd.Config.Targets == nil || len(cmd.Config.Targets) == 0 {
-		return nil, fmt.Errorf("no targets for download specified")
-	}
-
-	tmp := struct {
-		Targets Targets
-	}{}
-	err := yaml.Unmarshal(cmd.Config.Targets, &tmp)
-	if err != nil {
-		return nil, err
-	}
-	tgts := tmp.Targets
-
-	token := cmd.Credentials.Token
-	projectId := cmd.Config.DefaultProjectID
-	fileFormat := cmd.Config.DefaultFileFormat
-
-	validTargets := []*Target{}
-	for _, target := range tgts {
-		if target == nil {
-			continue
-		}
-		if target.ProjectID == "" {
-			target.ProjectID = projectId
-		}
-		if target.AccessToken == "" {
-			target.AccessToken = token
-		}
-		if target.FileFormat == "" {
-			target.FileFormat = fileFormat
-		}
-		validTargets = append(validTargets, target)
-	}
-
-	if len(validTargets) <= 0 {
-		return nil, fmt.Errorf("no targets could be identified! Refine the targets list in your config")
-	}
-
-	return validTargets, nil
-}
-
-func createFile(path string) error {
-	err := Exists(path)
-	if err != nil {
-		absDir := filepath.Dir(path)
-		err := Exists(absDir)
-		if err != nil {
-			os.MkdirAll(absDir, 0700)
-		}
-
-		f, err := os.Create(path)
-		if err != nil {
-			return err
-		}
-		defer f.Close()
-	}
-	return nil
+func pullMessage(localeFile *LocaleFile) {
```

Comment:
also ideally I would allow handing in a writer and not just do `fmt.Print` but rather `fmt.FPrintf(r, ...)` but not sure how that plays with the ct package